website/server/libs/cron.js

```diff
@@ -113,6 +113,87 @@ function performSleepTasks (user, tasksByType, now) {
   });
 }
 
+function trackCronAnalytics(analytics, user, _progress, options) {
+  if (!_progress) _progress = {down: 0, up: 0, collectedItems: 0};
+
+  analytics.track('Cron', {
+    category: 'behavior',
+    gaLabel: 'Cron Count',
+    gaValue: user.flags.cronCount,
+    uuid: user._id,
+    user,
+    resting: user.preferences.sleep,
+    cronCount: user.flags.cronCount,
+    progressUp: _.min([_progress.up, 900]),
+    progressDown: _progress.down,
+    headers: options.headers,
+    loginIncentives: user.loginIncentives,
+  });
+}
+
+
+function awardLoginIncentives (user) {
+  let notificationData = {};
+
+  switch (user.loginIncentives) {
+    case 1:
+      user.items.gear.owned.armor_special_bardRobes = true; // eslint-disable-line camelcase
+      break;
+    case 2:
+      user.purchased.background.incentiveBackgrounds = true;
+      break;
+    case 3:
+      user.items.gear.owned.head_special_bardHat = true; // eslint-disable-line camelcase
+      notificationData.reward = common.content.gear.flat.head_special_bardHat;
+      break;
+    case 4:
+      user.items.hatchingPotions.Purple = 1;
+      break;
+    case 5:
+      user.balance = 4 * 5;
+      break;
+    case 7:
+      user.items.quests.moon1 = 1;
+      break;
+    case 10:
+      user.items.hatchingPotions.Purple = 1;
+      break;
+    case 14:
+      user.balance = 4 * 3;
```

Comment:
this is going to override the entire balance, even if the user already had its own gems